cg/services/order_validation_service/workflows/tomte/validation_service.py

```diff
@@ -1,5 +1,23 @@
+from cg.services.order_validation_service.models.validation_error import ValidationErrors
+from cg.services.order_validation_service.order_validation_service import OrderValidationService
+from cg.services.order_validation_service.utils import apply_validation
+from cg.services.order_validation_service.workflows.tomte.validation.field_validator import (
+    TomteFieldValidator,
+)
+from cg.services.order_validation_service.workflows.tomte.validation_rules import (
+    TOMTE_VALIDATION_RULES,
+)
 
-class TomteOrderValidationService:
 
-    def __init__(self):
-        pass
+class TomteValidationService(OrderValidationService):
+
+    def __init__(self, field_validator: TomteFieldValidator):
+        self.field_validator = field_validator
```

Comment:
In case we could use the pydantic field validators we can omit this injection here and rely on just the model_validate_json() functionality if I am not mistaken? Would that be easier to maintain in the long run?

Reply:
That was our first approach! But we want to map the pydantic errors to a general error object and it was easier to extract it to a separate class